modules/virt-about-dr-methods.adoc

```diff
@@ -6,25 +6,19 @@
 [id="virt-about-dr-methods_{context}"]
 = About disaster recovery methods
 
-For an overview of disaster recovery (DR) concepts, architecture, and planning considerations, see the link:https://access.redhat.com/articles/7041594[Red Hat {VirtProductName} disaster recovery guide] in the Red Hat Knowledgebase.
+For an overview of disaster recovery (DR) concepts, architecture, and planning considerations, see the link:https://docs.redhat.com/articles/7041594[Red{nbsp}Hat {VirtProductName} disaster recovery guide] in the Red{nbsp}Hat Knowledgebase.
```

Comment:
Looks like KCS is not ported, so this would need to remain an access.redhat link.